Source/AliveLibAE/Fleech.cpp

```diff
@@ -1859,8 +1860,11 @@ void Fleech::InitTonguePolys_42B6E0()
 
 void Fleech::SetAnim_429D80()
 {
-    BYTE** ppRes = ResBlockForMotion_42A530(field_106_current_motion);
-    field_20_animation.Set_Animation_Data_409C80(sFleechFrameTableOffsets_5517E4[field_106_current_motion], ppRes);
+    //BYTE** ppRes = ResBlockForMotion_42A530(field_106_current_motion);
+    //field_20_animation.Set_Animation_Data_409C80(sFleechFrameTableOffsets_5517E4[field_106_current_motion], ppRes);
+	
+	const AnimRecord& animRec = AnimRec(sFleechFrameTableOffsets_5517E4[field_106_current_motion]);
+    field_20_animation.Set_Animation_Data_409C80(animRec.mFrameTableOffset, nullptr);
```

Comment:
I'm not sure this can work because ppRes value wouldn't always be the same from ResBlockForMotion_42A530 depending on what field_106_current_motion's value was